packages/x-internal-gestures/src/core/gestures/TurnWheelGesture.ts

```diff
@@ -321,6 +321,7 @@ export class TurnWheelGesture<GestureName extends string> extends Gesture<Gestur
     const domEvent = new CustomEvent(eventName, {
       bubbles: true,
       cancelable: true,
+      composed: true,
```

Comment:
Why do we need this?

Reply:
They currently follow the native event. All of them have `composed:true`, except some special events.

`composed:true` allows events triggered by an element inside shadow-dom to bubble up to elements outside the shadow dom